ext/drpttracer/drpttracer.c

```diff
@@ -556,7 +529,7 @@ drpttracer_end_tracing(IN void *drcontext, INOUT void *tracer_handle,
     if (handle->header->aux_head > handle->header->aux_size) {
         ERRMSG("the buffer is full and the new PT data is overwritten\n");
         ASSERT(false, "aux_head > aux_size");
-        return DRPTTRACER_ERROR_FAILED_TO_READ_TRACE;
+        return DRPTTRACER_ERROR_OVERWRITTEEN_PT_TRACE;
```

Comment:
s/DRPTTRACER_ERROR_OVERWRITTEEN_PT_TRACE/DRPTTRACER_ERROR_OVERWRITTEN_PT_TRACE/

Reply:
And in dox too.

Reply:
Done.